### ISSUE TITLE
S.M:A:R:T Devices Empty (6.x)

### ISSUE THREAD

**Issue by long-walk** [openmediavault/openmediavault#1054]
**Describe the bug**

The menu entry "s.m.a.r.t Devices" shows 4 rows but they not filed with content and if I select one and put info or edit I got an error 404.

I go trough all logs in /var/logs  but couldn't find any error or hint what causing this error.
If I press refresh I see ruing Processes smartctl, lvdisplay, php-fpm7.4, omv-engined  so it do something.

Also I have to note that I use a not recommended Device Thecus N5200 i686

**To Reproduce**

1. Select S:M:A:R:T Devices from the menue

**Expected behavior**
Shows the Data in the rows

**Screenshots**

Show Data
![omv1](https://user-images.githubusercontent.com/56696022/123559747-43f70980-d79e-11eb-89b8-7ef37a159366.jpg)

- OS version: [uname -a]
Linux NAS 5.10.0-7-686 #1 SMP Debian 5.10.40-1 (2021-05-28) i686 GNU/Linux
ii  openmediavault                 6.0-10                         all          openmediavault - The open network attached storage solution
ii  openmediavault-diskstats       6.0-2                          all          openmediavault disk monitoring plugin
ii  openmediavault-keyring         1.0                            all          GnuPG archive keys of the OpenMediaVault archive

**Client (please complete the following information):**

- Device: PC
- OS: Fedora 33
- Browser: Self compiled Firefox  
- 91.0a1 (2021-06-26) (64-bit)

**Comment by long-walk** [comment]
OMV Report
[omv_report.log](https://github.com/openmediavault/openmediavault/files/6722343/omv_report.log)

**Comment by votdev** [comment]
Couldn't reproduce this on Firefox 89.0.1 and Chrome 91.0.4472.114.

![Bildschirmfoto vom 2021-06-28 08-53-37](https://user-images.githubusercontent.com/1897962/123593105-6c0e5900-d7ee-11eb-863b-ecfd04a128f9.png)

Please try another browser to check if it is a problem of your self-compiler Firefox.

Could you please run `omv-rpc -u admin 'Smart' 'getList' '{"start":0,"limit":-1}' | jq` in the CLI and post the output here?

**Comment by long-walk** [comment]
Have the same problem with Fedora Standard Firefox 89.0.2 .
root@NAS:~# omv-rpc -u admin 'Smart' 'getList' '{"start":0,"limit":-1}' | jq
{
  "total": 4,
  "data": {
    "3": {
      "devicename": "sda",
      "canonicaldevicefile": "/dev/sda",
      "devicefile": "/dev/disk/by-id/ata-HYPERDISK_SSD_190800008335",
      "devicelinks": [
        "/dev/disk/by-id/ata-HYPERDISK_SSD_190800008335",
        "/dev/disk/by-path/pci-0000:00:1f.1-ata-1",
        "/dev/disk/by-path/pci-0000:00:1f.1-ata-1.0"
      ],
      "model": "HYPERDISK SSD",
      "size": "7708925952",
      "temperature": "50°C",
      "description": "HYPERDISK SSD [/dev/sda, 7.17 GiB]",
      "vendor": "ATA",
      "serialnumber": "190800008335",
      "overallstatus": "BAD_SECTOR",
      "uuid": "fa4b1c66-ef79-11e5-87a0-0002b3a176b4",
      "monitor": false
    },
    "4": {
      "devicename": "sdb",
      "canonicaldevicefile": "/dev/sdb",
      "devicefile": "/dev/disk/by-id/ata-WDC_WD2500KS-00MJB0_WD-WCANKH354513",
      "devicelinks": [
        "/dev/disk/by-id/ata-WDC_WD2500KS-00MJB0_WD-WCANKH354513",
        "/dev/disk/by-path/pci-0000:01:00.0-ata-1",
        "/dev/disk/by-path/pci-0000:01:00.0-ata-1.0"
      ],
      "model": "WDC WD2500KS-00MJB0",
      "size": "250059350016",
      "temperature": "34°C",
      "description": "WDC WD2500KS-00MJB0 [/dev/sdb, 232.88 GiB]",
      "vendor": "ATA",
      "serialnumber": "WD-WCANKH354513",
      "overallstatus": "GOOD",
      "uuid": "fa4b1c66-ef79-11e5-87a0-0002b3a176b4",
      "monitor": false
    },
    "5": {
      "devicename": "sdc",
      "canonicaldevicefile": "/dev/sdc",
      "devicefile": "/dev/disk/by-id/ata-WDC_WD2500KS-00MJB0_WD-WCANKJ162951",
      "devicelinks": [
        "/dev/disk/by-id/ata-WDC_WD2500KS-00MJB0_WD-WCANKJ162951",
        "/dev/disk/by-path/pci-0000:01:00.0-ata-3",
        "/dev/disk/by-path/pci-0000:01:00.0-ata-3.0"
      ],
      "model": "WDC WD2500KS-00MJB0",
      "size": "250059350016",
      "temperature": "28°C",
      "description": "WDC WD2500KS-00MJB0 [/dev/sdc, 232.88 GiB]",
      "vendor": "ATA",
      "serialnumber": "WD-WCANKJ162951",
      "overallstatus": "GOOD",
      "uuid": "fa4b1c66-ef79-11e5-87a0-0002b3a176b4",
      "monitor": false
    },
    "6": {
      "devicename": "sdd",
      "canonicaldevicefile": "/dev/sdd",
      "devicefile": "/dev/disk/by-id/ata-WDC_WD2500KS-00MJB0_WD-WCANKJ163061",
      "devicelinks": [
        "/dev/disk/by-id/ata-WDC_WD2500KS-00MJB0_WD-WCANKJ163061",
        "/dev/disk/by-path/pci-0000:01:00.0-ata-2",
        "/dev/disk/by-path/pci-0000:01:00.0-ata-2.0"
      ],
      "model": "WDC WD2500KS-00MJB0",
      "size": "250059350016",
      "temperature": "30°C",
      "description": "WDC WD2500KS-00MJB0 [/dev/sdd, 232.88 GiB]",
      "vendor": "ATA",
      "serialnumber": "WD-WCANKJ163061",
      "overallstatus": "GOOD",
      "uuid": "fa4b1c66-ef79-11e5-87a0-0002b3a176b4",
      "monitor": false
    }
  }
}

**Comment by votdev** [comment]
This is not a UI problem, the RPC request provides corrupted data. Need to investigate that.

**Comment by votdev** [comment]
Could you please post the ouput of `omv-rpc -u admin 'Smart' 'enumerateDevices' | jq`.

**Comment by long-walk** [comment]
root@NAS:~# omv-rpc -u admin 'Smart' 'enumerateDevices' | jq
{
  "3": {
    "devicename": "sda",
    "canonicaldevicefile": "/dev/sda",
    "devicefile": "/dev/disk/by-id/ata-HYPERDISK_SSD_190800008335",
    "devicelinks": [
      "/dev/disk/by-id/ata-HYPERDISK_SSD_190800008335",
      "/dev/disk/by-path/pci-0000:00:1f.1-ata-1",
      "/dev/disk/by-path/pci-0000:00:1f.1-ata-1.0"
    ],
    "model": "HYPERDISK SSD",
    "size": "7708925952",
    "temperature": "43°C",
    "description": "HYPERDISK SSD [/dev/sda, 7.17 GiB]",
    "vendor": "ATA",
    "serialnumber": "190800008335",
    "overallstatus": "BAD_SECTOR"
  },
  "4": {
    "devicename": "sdb",
    "canonicaldevicefile": "/dev/sdb",
    "devicefile": "/dev/disk/by-id/ata-WDC_WD2500KS-00MJB0_WD-WCANKH354513",
    "devicelinks": [
      "/dev/disk/by-id/ata-WDC_WD2500KS-00MJB0_WD-WCANKH354513",
      "/dev/disk/by-path/pci-0000:01:00.0-ata-1",
      "/dev/disk/by-path/pci-0000:01:00.0-ata-1.0"
    ],
    "model": "WDC WD2500KS-00MJB0",
    "size": "250059350016",
    "temperature": "34°C",
    "description": "WDC WD2500KS-00MJB0 [/dev/sdb, 232.88 GiB]",
    "vendor": "ATA",
    "serialnumber": "WD-WCANKH354513",
    "overallstatus": "GOOD"
  },
  "5": {
    "devicename": "sdc",
    "canonicaldevicefile": "/dev/sdc",
    "devicefile": "/dev/disk/by-id/ata-WDC_WD2500KS-00MJB0_WD-WCANKJ162951",
    "devicelinks": [
      "/dev/disk/by-id/ata-WDC_WD2500KS-00MJB0_WD-WCANKJ162951",
      "/dev/disk/by-path/pci-0000:01:00.0-ata-3",
      "/dev/disk/by-path/pci-0000:01:00.0-ata-3.0"
    ],
    "model": "WDC WD2500KS-00MJB0",
    "size": "250059350016",
    "temperature": "28°C",
    "description": "WDC WD2500KS-00MJB0 [/dev/sdc, 232.88 GiB]",
    "vendor": "ATA",
    "serialnumber": "WD-WCANKJ162951",
    "overallstatus": "GOOD"
  },
  "6": {
    "devicename": "sdd",
    "canonicaldevicefile": "/dev/sdd",
    "devicefile": "/dev/disk/by-id/ata-WDC_WD2500KS-00MJB0_WD-WCANKJ163061",
    "devicelinks": [
      "/dev/disk/by-id/ata-WDC_WD2500KS-00MJB0_WD-WCANKJ163061",
      "/dev/disk/by-path/pci-0000:01:00.0-ata-2",
      "/dev/disk/by-path/pci-0000:01:00.0-ata-2.0"
    ],
    "model": "WDC WD2500KS-00MJB0",
    "size": "250059350016",
    "temperature": "30°C",
    "description": "WDC WD2500KS-00MJB0 [/dev/sdd, 232.88 GiB]",
    "vendor": "ATA",
    "serialnumber": "WD-WCANKJ163061",
    "overallstatus": "GOOD"
  }
}

**Comment by votdev** [comment]
Is it possible that your system has 3 or more disks that do not support SMART? I think so, and this is the reason for your problems. You hit a bug.

**Comment by long-walk** [comment]
Thanks for your analyses Yes that possible I think the System disk module maybe has no smart.

**Comment by long-walk** [comment]
Unfortunately all disk have SMART support but one device is not in smartctl Database

root@NAS:~# smartctl -a /dev/sda
smartctl 7.2 2020-12-30 r5155 [i686-linux-5.10.0-7-686] (local build)
Copyright (C) 2002-20, Bruce Allen, Christian Franke, www.smartmontools.org

=== START OF INFORMATION SECTION ===
Device Model:     HYPERDISK  SSD
Serial Number:    190800008335
Firmware Version: 20150807
User Capacity:    7,708,925,952 bytes [7.70 GB]
Sector Size:      512 bytes logical/physical
Rotation Rate:    Solid State Device
TRIM Command:     Unavailable
Device is:        Not in smartctl database [for details use: -P showall]
ATA Version is:   ACS-2 (minor revision not indicated)
Local Time is:    Tue Jun 29 21:26:27 2021 CEST
SMART support is: Available - device has SMART capability.
SMART support is: Enabled




```bash
boot@NAS:~#` fdisk -l
Disk /dev/sda: 7.18 GiB, 7708925952 bytes, 15056496 sectors
Disk model: HYPERDISK  SSD  
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: dos
Disk identifier: 0xa971847e

Device     Boot Start      End  Sectors  Size Id Type
/dev/sda1  *     2048 15054847 15052800  7.2G 83 Linux


Disk /dev/sdb: 232.89 GiB, 250059350016 bytes, 488397168 sectors
Disk model: WDC WD2500KS-00M
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: dos
Disk identifier: 0x37a630ed

Device     Boot   Start       End   Sectors  Size Id Type
/dev/sdb1            63   3919859   3919797  1.9G fd Linux raid autodetect
/dev/sdb2       3919860 488392064 484472205  231G fd Linux raid autodetect


Disk /dev/sdc: 232.89 GiB, 250059350016 bytes, 488397168 sectors
Disk model: WDC WD2500KS-00M
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: dos
Disk identifier: 0x20143c25

Device     Boot   Start       End   Sectors  Size Id Type
/dev/sdc1            63   3919859   3919797  1.9G fd Linux raid autodetect
/dev/sdc2       3919860 488392064 484472205  231G fd Linux raid autodetect


Disk /dev/sdd: 232.89 GiB, 250059350016 bytes, 488397168 sectors
Disk model: WDC WD2500KS-00M
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: dos
Disk identifier: 0x1882a2a5

Device     Boot   Start       End   Sectors  Size Id Type
/dev/sdd1            63   3919859   3919797  1.9G fd Linux raid autodetect
/dev/sdd2       3919860 488392064 484472205  231G fd Linux raid autodetect


Disk /dev/md127: 462.03 GiB, 496099393536 bytes, 968944128 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 65536 bytes / 131072 bytes


Disk /dev/mapper/vg0-syslv: 1 GiB, 1073741824 bytes, 2097152 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 65536 bytes / 131072 bytes


Disk /dev/mapper/vg0-lv0: 437.9 GiB, 470194061312 bytes, 918347776 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 65536 bytes / 131072 bytes


Disk /dev/mapper/vg0-lv1: 23.1 GiB, 24805113856 bytes, 48447488 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 65536 bytes / 131072 bytes
root@NAS:~# smart
smartctl  smartd    
root@NAS:~# smartctl /dev/sda
smartctl 7.2 2020-12-30 r5155 [i686-linux-5.10.0-7-686] (local build)
Copyright (C) 2002-20, Bruce Allen, Christian Franke, www.smartmontools.org

ATA device successfully opened

Use 'smartctl -a' (or '-x') to print SMART (and more) information

root@NAS:~# smartctl -a /dev/sda
smartctl 7.2 2020-12-30 r5155 [i686-linux-5.10.0-7-686] (local build)
Copyright (C) 2002-20, Bruce Allen, Christian Franke, www.smartmontools.org

=== START OF INFORMATION SECTION ===
Device Model:     HYPERDISK  SSD
Serial Number:    190800008335
Firmware Version: 20150807
User Capacity:    7,708,925,952 bytes [7.70 GB]
Sector Size:      512 bytes logical/physical
Rotation Rate:    Solid State Device
TRIM Command:     Unavailable
Device is:        Not in smartctl database [for details use: -P showall]
ATA Version is:   ACS-2 (minor revision not indicated)
Local Time is:    Tue Jun 29 21:26:27 2021 CEST
SMART support is: Available - device has SMART capability.
SMART support is: Enabled

=== START OF READ SMART DATA SECTION ===
SMART overall-health self-assessment test result: PASSED

General SMART Values:
Offline data collection status:  (0x00) Offline data collection activity
                                        was never started.
                                        Auto Offline Data Collection: Disabled.
Total time to complete Offline 
data collection:                (    0) seconds.
Offline data collection
capabilities:                    (0x00)         Offline data collection not supported.
SMART capabilities:            (0x0002) Does not save SMART data before
                                        entering power-saving mode.
                                        Supports SMART auto save timer.
Error logging capability:        (0x00) Error logging NOT supported.
                                        No General Purpose Logging support.

SMART Attributes Data Structure revision number: 1
Vendor Specific SMART Attributes with Thresholds:
ID# ATTRIBUTE_NAME          FLAG     VALUE WORST THRESH TYPE      UPDATED  WHEN_FAILED RAW_VALUE
  1 Raw_Read_Error_Rate     0x0000   100   100   050    Old_age   Offline      -       0
  5 Reallocated_Sector_Ct   0x0000   100   100   050    Old_age   Offline      -       1
 12 Power_Cycle_Count       0x0000   100   100   050    Old_age   Offline      -       97
160 Unknown_Attribute       0x0000   100   100   050    Old_age   Offline      -       8
161 Unknown_Attribute       0x0000   100   100   050    Old_age   Offline      -       34
162 Unknown_Attribute       0x0000   100   100   050    Old_age   Offline      -       21
163 Unknown_Attribute       0x0000   100   100   050    Old_age   Offline      -       11
164 Unknown_Attribute       0x0000   100   100   050    Old_age   Offline      -       18229
165 Unknown_Attribute       0x0000   100   100   050    Old_age   Offline      -       71
166 Unknown_Attribute       0x0000   100   100   050    Old_age   Offline      -       3
167 Unknown_Attribute       0x0000   100   100   050    Old_age   Offline      -       36
192 Power-Off_Retract_Count 0x0000   100   100   050    Old_age   Offline      -       16
194 Temperature_Celsius     0x0000   100   100   000    Old_age   Offline      -       46 (Min/Max 32/67)
195 Hardware_ECC_Recovered  0x0000   100   100   050    Old_age   Offline      -       373670
196 Reallocated_Event_Count 0x0000   100   100   050    Old_age   Offline      -       249
198 Offline_Uncorrectable   0x0000   100   100   050    Old_age   Offline      -       352
199 UDMA_CRC_Error_Count    0x0000   100   100   050    Old_age   Offline      -       2
241 Total_LBAs_Written      0x0000   100   100   050    Old_age   Offline      -       1543
242 Total_LBAs_Read         0x0000   100   100   050    Old_age   Offline      -       2058

SMART Error Log not supported

SMART Self-test Log not supported

Selective Self-tests/Logging not supported

root@NAS:~# smartctl -a /dev/sdb
smartctl 7.2 2020-12-30 r5155 [i686-linux-5.10.0-7-686] (local build)
Copyright (C) 2002-20, Bruce Allen, Christian Franke, www.smartmontools.org

=== START OF INFORMATION SECTION ===
Model Family:     Western Digital Caviar SE16 Serial ATA
Device Model:     WDC WD2500KS-00MJB0
Serial Number:    WD-WCANKH354513
Firmware Version: 02.01C03
User Capacity:    250,059,350,016 bytes [250 GB]
Sector Size:      512 bytes logical/physical
Device is:        In smartctl database [for details use: -P show]
ATA Version is:   ATA/ATAPI-7 (minor revision not indicated)
Local Time is:    Tue Jun 29 21:26:31 2021 CEST
SMART support is: Available - device has SMART capability.
SMART support is: Enabled

=== START OF READ SMART DATA SECTION ===
SMART overall-health self-assessment test result: PASSED
See vendor-specific Attribute list for marginal Attributes.

General SMART Values:
Offline data collection status:  (0x82) Offline data collection activity
                                        was completed without error.
                                        Auto Offline Data Collection: Enabled.
Self-test execution status:      (   0) The previous self-test routine completed
                                        without error or no self-test has ever 
                                        been run.
Total time to complete Offline 
data collection:                ( 7680) seconds.
Offline data collection
capabilities:                    (0x7b) SMART execute Offline immediate.
                                        Auto Offline data collection on/off support.
                                        Suspend Offline collection upon new
                                        command.
                                        Offline surface scan supported.
                                        Self-test supported.
                                        Conveyance Self-test supported.
                                        Selective Self-test supported.
SMART capabilities:            (0x0003) Saves SMART data before entering
                                        power-saving mode.
                                        Supports SMART auto save timer.
Error logging capability:        (0x01) Error logging supported.
                                        General Purpose Logging supported.
Short self-test routine 
recommended polling time:        (   2) minutes.
Extended self-test routine
recommended polling time:        (  90) minutes.
Conveyance self-test routine
recommended polling time:        (   6) minutes.
SCT capabilities:              (0x103f) SCT Status supported.
                                        SCT Error Recovery Control supported.
                                        SCT Feature Control supported.
                                        SCT Data Table supported.

SMART Attributes Data Structure revision number: 16
Vendor Specific SMART Attributes with Thresholds:
ID# ATTRIBUTE_NAME          FLAG     VALUE WORST THRESH TYPE      UPDATED  WHEN_FAILED RAW_VALUE
  1 Raw_Read_Error_Rate     0x000f   200   200   051    Pre-fail  Always       -       0
  3 Spin_Up_Time            0x0003   191   189   021    Pre-fail  Always       -       5450
  4 Start_Stop_Count        0x0032   100   100   000    Old_age   Always       -       755
  5 Reallocated_Sector_Ct   0x0033   200   200   140    Pre-fail  Always       -       0
  7 Seek_Error_Rate         0x000f   200   200   051    Pre-fail  Always       -       0
  9 Power_On_Hours          0x0032   100   100   000    Old_age   Always       -       630
 10 Spin_Retry_Count        0x0013   100   100   051    Pre-fail  Always       -       0
 11 Calibration_Retry_Count 0x0012   100   100   051    Old_age   Always       -       0
 12 Power_Cycle_Count       0x0032   100   100   000    Old_age   Always       -       453
190 Airflow_Temperature_Cel 0x0022   064   042   045    Old_age   Always   In_the_past 36
194 Temperature_Celsius     0x0022   114   092   000    Old_age   Always       -       36
196 Reallocated_Event_Count 0x0032   200   200   000    Old_age   Always       -       0
197 Current_Pending_Sector  0x0012   200   200   000    Old_age   Always       -       0
198 Offline_Uncorrectable   0x0010   200   200   000    Old_age   Offline      -       0
199 UDMA_CRC_Error_Count    0x003e   200   200   000    Old_age   Always       -       0
200 Multi_Zone_Error_Rate   0x0009   200   200   051    Pre-fail  Offline      -       0

SMART Error Log Version: 1
No Errors Logged

SMART Self-test log structure revision number 1
No self-tests have been logged.  [To run self-tests, use: smartctl -t]

SMART Selective self-test log data structure revision number 1
 SPAN  MIN_LBA  MAX_LBA  CURRENT_TEST_STATUS
    1        0        0  Not_testing
    2        0        0  Not_testing
    3        0        0  Not_testing
    4        0        0  Not_testing
    5        0        0  Not_testing
Selective self-test flags (0x0):
  After scanning selected spans, do NOT read-scan remainder of disk.
If Selective self-test is pending on power-up, resume after 0 minute delay.

root@NAS:~# smartctl -a /dev/sdc
smartctl 7.2 2020-12-30 r5155 [i686-linux-5.10.0-7-686] (local build)
Copyright (C) 2002-20, Bruce Allen, Christian Franke, www.smartmontools.org

=== START OF INFORMATION SECTION ===
Model Family:     Western Digital Caviar SE16 Serial ATA
Device Model:     WDC WD2500KS-00MJB0
Serial Number:    WD-WCANKJ162951
Firmware Version: 02.01C03
User Capacity:    250,059,350,016 bytes [250 GB]
Sector Size:      512 bytes logical/physical
Device is:        In smartctl database [for details use: -P show]
ATA Version is:   ATA/ATAPI-7 (minor revision not indicated)
Local Time is:    Tue Jun 29 21:26:35 2021 CEST
SMART support is: Available - device has SMART capability.
SMART support is: Enabled

=== START OF READ SMART DATA SECTION ===
SMART overall-health self-assessment test result: PASSED
See vendor-specific Attribute list for marginal Attributes.

General SMART Values:
Offline data collection status:  (0x82) Offline data collection activity
                                        was completed without error.
                                        Auto Offline Data Collection: Enabled.
Self-test execution status:      (   0) The previous self-test routine completed
                                        without error or no self-test has ever 
                                        been run.
Total time to complete Offline 
data collection:                ( 8280) seconds.
Offline data collection
capabilities:                    (0x7b) SMART execute Offline immediate.
                                        Auto Offline data collection on/off support.
                                        Suspend Offline collection upon new
                                        command.
                                        Offline surface scan supported.
                                        Self-test supported.
                                        Conveyance Self-test supported.
                                        Selective Self-test supported.
SMART capabilities:            (0x0003) Saves SMART data before entering
                                        power-saving mode.
                                        Supports SMART auto save timer.
Error logging capability:        (0x01) Error logging supported.
                                        General Purpose Logging supported.
Short self-test routine 
recommended polling time:        (   2) minutes.
Extended self-test routine
recommended polling time:        (  96) minutes.
Conveyance self-test routine
recommended polling time:        (   6) minutes.
SCT capabilities:              (0x103f) SCT Status supported.
                                        SCT Error Recovery Control supported.
                                        SCT Feature Control supported.
                                        SCT Data Table supported.

SMART Attributes Data Structure revision number: 16
Vendor Specific SMART Attributes with Thresholds:
ID# ATTRIBUTE_NAME          FLAG     VALUE WORST THRESH TYPE      UPDATED  WHEN_FAILED RAW_VALUE
  1 Raw_Read_Error_Rate     0x000f   200   200   051    Pre-fail  Always       -       0
  3 Spin_Up_Time            0x0003   186   186   021    Pre-fail  Always       -       5658
  4 Start_Stop_Count        0x0032   100   100   000    Old_age   Always       -       752
  5 Reallocated_Sector_Ct   0x0033   200   200   140    Pre-fail  Always       -       0
  7 Seek_Error_Rate         0x000f   200   200   051    Pre-fail  Always       -       0
  9 Power_On_Hours          0x0032   100   100   000    Old_age   Always       -       621
 10 Spin_Retry_Count        0x0013   100   100   051    Pre-fail  Always       -       0
 11 Calibration_Retry_Count 0x0012   100   100   051    Old_age   Always       -       0
 12 Power_Cycle_Count       0x0032   100   100   000    Old_age   Always       -       453
190 Airflow_Temperature_Cel 0x0022   069   040   045    Old_age   Always   In_the_past 31
194 Temperature_Celsius     0x0022   119   090   000    Old_age   Always       -       31
196 Reallocated_Event_Count 0x0032   200   200   000    Old_age   Always       -       0
197 Current_Pending_Sector  0x0012   200   200   000    Old_age   Always       -       0
198 Offline_Uncorrectable   0x0010   200   200   000    Old_age   Offline      -       0
199 UDMA_CRC_Error_Count    0x003e   200   200   000    Old_age   Always       -       0
200 Multi_Zone_Error_Rate   0x0009   200   200   051    Pre-fail  Offline      -       0

SMART Error Log Version: 1
No Errors Logged

SMART Self-test log structure revision number 1
No self-tests have been logged.  [To run self-tests, use: smartctl -t]

SMART Selective self-test log data structure revision number 1
 SPAN  MIN_LBA  MAX_LBA  CURRENT_TEST_STATUS
    1        0        0  Not_testing
    2        0        0  Not_testing
    3        0        0  Not_testing
    4        0        0  Not_testing
    5        0        0  Not_testing
Selective self-test flags (0x0):
  After scanning selected spans, do NOT read-scan remainder of disk.
If Selective self-test is pending on power-up, resume after 0 minute delay.

root@NAS:~# smartctl -a /dev/sdd
smartctl 7.2 2020-12-30 r5155 [i686-linux-5.10.0-7-686] (local build)
Copyright (C) 2002-20, Bruce Allen, Christian Franke, www.smartmontools.org

=== START OF INFORMATION SECTION ===
Model Family:     Western Digital Caviar SE16 Serial ATA
Device Model:     WDC WD2500KS-00MJB0
Serial Number:    WD-WCANKJ163061
Firmware Version: 02.01C03
User Capacity:    250,059,350,016 bytes [250 GB]
Sector Size:      512 bytes logical/physical
Device is:        In smartctl database [for details use: -P show]
ATA Version is:   ATA/ATAPI-7 (minor revision not indicated)
Local Time is:    Tue Jun 29 21:26:41 2021 CEST
SMART support is: Available - device has SMART capability.
SMART support is: Enabled

=== START OF READ SMART DATA SECTION ===
SMART overall-health self-assessment test result: PASSED
See vendor-specific Attribute list for marginal Attributes.

General SMART Values:
Offline data collection status:  (0x82) Offline data collection activity
                                        was completed without error.
                                        Auto Offline Data Collection: Enabled.
Self-test execution status:      (   0) The previous self-test routine completed
                                        without error or no self-test has ever 
                                        been run.
Total time to complete Offline 
data collection:                ( 7680) seconds.
Offline data collection
capabilities:                    (0x7b) SMART execute Offline immediate.
                                        Auto Offline data collection on/off support.
                                        Suspend Offline collection upon new
                                        command.
                                        Offline surface scan supported.
                                        Self-test supported.
                                        Conveyance Self-test supported.
                                        Selective Self-test supported.
SMART capabilities:            (0x0003) Saves SMART data before entering
                                        power-saving mode.
                                        Supports SMART auto save timer.
Error logging capability:        (0x01) Error logging supported.
                                        General Purpose Logging supported.
Short self-test routine 
recommended polling time:        (   2) minutes.
Extended self-test routine
recommended polling time:        (  90) minutes.
Conveyance self-test routine
recommended polling time:        (   6) minutes.
SCT capabilities:              (0x103f) SCT Status supported.
                                        SCT Error Recovery Control supported.
                                        SCT Feature Control supported.
                                        SCT Data Table supported.

SMART Attributes Data Structure revision number: 16
Vendor Specific SMART Attributes with Thresholds:
ID# ATTRIBUTE_NAME          FLAG     VALUE WORST THRESH TYPE      UPDATED  WHEN_FAILED RAW_VALUE
  1 Raw_Read_Error_Rate     0x000f   200   200   051    Pre-fail  Always       -       0
  3 Spin_Up_Time            0x0003   185   183   021    Pre-fail  Always       -       5708
  4 Start_Stop_Count        0x0032   100   100   000    Old_age   Always       -       757
  5 Reallocated_Sector_Ct   0x0033   200   200   140    Pre-fail  Always       -       0
  7 Seek_Error_Rate         0x000f   200   200   051    Pre-fail  Always       -       0
  9 Power_On_Hours          0x0032   100   100   000    Old_age   Always       -       629
 10 Spin_Retry_Count        0x0013   100   100   051    Pre-fail  Always       -       0
 11 Calibration_Retry_Count 0x0012   100   100   051    Old_age   Always       -       0
 12 Power_Cycle_Count       0x0032   100   100   000    Old_age   Always       -       453
190 Airflow_Temperature_Cel 0x0022   067   036   045    Old_age   Always   In_the_past 33
194 Temperature_Celsius     0x0022   117   086   000    Old_age   Always       -       33
196 Reallocated_Event_Count 0x0032   200   200   000    Old_age   Always       -       0
197 Current_Pending_Sector  0x0012   200   200   000    Old_age   Always       -       0
198 Offline_Uncorrectable   0x0010   200   200   000    Old_age   Offline      -       0
199 UDMA_CRC_Error_Count    0x003e   200   200   000    Old_age   Always       -       0
200 Multi_Zone_Error_Rate   0x0009   200   200   051    Pre-fail  Offline      -       0

SMART Error Log Version: 1
No Errors Logged

SMART Self-test log structure revision number 1
No self-tests have been logged.  [To run self-tests, use: smartctl -t]

SMART Selective self-test log data structure revision number 1
 SPAN  MIN_LBA  MAX_LBA  CURRENT_TEST_STATUS
    1        0        0  Not_testing
    2        0        0  Not_testing
    3        0        0  Not_testing
    4        0        0  Not_testing
    5        0        0  Not_testing
Selective self-test flags (0x0):
  After scanning selected spans, do NOT read-scan remainder of disk.
If Selective self-test is pending on power-up, resume after 0 minute delay.
```

**Comment by votdev** [comment]
Thx for the additional info but this was not necessary. I think the issue will be fixed with the mentioned commit above.

Please report back when the new package is tested.

**Comment by long-walk** [comment]
Problem is fixed with 6.0-12. Thanks
